Justfile

```diff
@@ -0,0 +1,10 @@
+# List available commands
+_default:
+    just --list
+
+sqlx-prepare:
+  cargo sqlx prepare \
+    --database-url $DOCSRS_DATABASE_URL \
+    --workspace \
+    --check \
```

Comment:
Should this flag be removed so this actually updates the files (and maybe add another `sqlx-check` command that runs it?).

Reply:
that was a copy / paste error, thanks!